### PR TITLE
Improve fixup-libgfortran.sh

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: contrib/fixup-libgfortran.sh [hunk 1]
```diff
@@ -82,23 +82,23 @@ echo "Found traces of libgfortran/libgcc in $LIBGFORTRAN_DIRS"
 
 # Do the private_libdir libraries
 if [ "$UNAME" = "Darwin" ]; then
-    cd $private_libdir
+    cd "$private_libdir" || exit
```
Comment (yuyichao) [inline contrib/fixup-libgfortran.sh]: Should not return `0`.

File: contrib/fixup-libgfortran.sh [hunk 2]
```diff
@@ -82,23 +82,23 @@ echo "Found traces of libgfortran/libgcc in $LIBGFORTRAN_DIRS"
...
+                install_name_tool -change "$dir/libquadmath.0.dylib" @rpath/libquadmath.0.dylib "$dylib"
+                install_name_tool -change "$dir/libgcc_s.1.dylib" @rpath/libgcc_s.1.dylib "$dylib"
             done
         done
     done
 fi
 
 if [ "$UNAME" = "Linux" ]; then
-    cd $private_libdir
+    cd "$private_libdir" || exit
```
Comment (yuyichao) [inline contrib/fixup-libgfortran.sh]: Should not return `0`.

**Review by yuyichao — CHANGES_REQUESTED** [review]

**Comment by yuyichao** [comment]
LGTM otherwise.

**Comment by pkofod** [comment]
Comments seem addressed, right?

**Comment by tkelman** [comment]
does this fix any real problems?

**Comment by Keno** [comment]
Original repository was deleted, so this PR can't be rebased. Any remaining issues can be addressed in a new PR.

**Pull request closed without merging**